kvmagent/kvmagent/plugins/vm_plugin.py

```
@@ -13525,6 +13593,75 @@ def _set_vf_mac(pci_addr, mac):
+        return jsonobject.dumps(rsp)
+
+    @kvmagent.replyerror
+    def resolve_vtpm_libvirt_secret_uuid(self, req):
+        cmd = jsonobject.loads(req[http.REQUEST_BODY])
+        rsp = kvmagent.AgentResponse()
+        domain_xml = None
+        try:
+            conn = LibvirtSingleton.get_connection()
+            dom = conn.lookupByName(cmd.vmInstanceUuid)
+            domain_xml = dom.XMLDesc(0)
+        except Exception as ex:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)